src/include/execution/sql/value.h

```diff
@@ -237,38 +179,54 @@ struct StringVal : public Val {
   std::string_view StringView() const {
     TERRIER_ASSERT(!is_null_,
                    "You should be doing a NULL check before attempting to generate a std::string_view of a StringVal.");
-    return std::string_view(Content(), len_);
+    return std::string_view(GetContent(), GetLength());
```

Comment:
I think we can simplify this to return `VarlenEntry`'s underlying `StringView()`.